homeassistant/components/epson/media_player.py

```diff
@@ -114,8 +108,11 @@ async def async_update(self):
         """Update state of device."""
         power_state = await self._projector.get_property(POWER)
         _LOGGER.debug("Projector status: %s", power_state)
-        if not power_state:
+        if not power_state or power_state == EPSON_STATE_UNAVAILABLE:
+            self._state = False
```

Comment:
The state itself should not change. The thing is; when a device becomes unavailable, we actually don't know the state of the device (that is why it is unavailable in the first place).

Reply:
got it. removed this line.